src/js/content/checkCSPForEvals.ts

```diff
@@ -80,10 +73,10 @@ function getIsValidScriptSrcAndHasScriptSrcDirective(
   return [isValidScriptSrc, hasScriptSrcDirective];
 }
 
-export function checkCSPHeaders(
+export function checkCSPForEvals(
   cspHeaders: Array<string>,
   cspReportHeaders: Array<string>,
-): boolean {
```

Comment:
Did you mean to get rid of this return type?

Reply:
Unintentional, wil fix.